pkg/server/bootstrap/identity.go

```diff
@@ -198,8 +199,14 @@ func wildcardIdentitiesResolver(ids *identities,
 	}
 }
 
-func apiExportIdentityProvider(rootShardKcpClient kcpclient.Interface, localShardKubeClusterClient kubernetes.ClusterInterface) func(ctx context.Context, apiExportName string) (string, error) {
+func apiExportIdentityProvider(config *rest.Config, localShardKubeClusterClient kubernetes.ClusterInterface) func(ctx context.Context, apiExportName string) (string, error) {
```

Comment:
BTW - any reason we're not getting the `kubernetes.ClusterInterface` here?

Reply:
Need https://github.com/kcp-dev/kcp/pull/1753 to get in to scope this.

Reply:
Will do it a follow up after its merged.